src/applications/edu-benefits/0993/containers/SubmitButtons.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import moment from 'moment';
+import ProgressButton from 'us-forms-system/lib/js/components/ProgressButton';
+import { timeFromNow } from 'us-forms-system/lib/js/utilities/date';
+
+export default function SubmitButtons(props) {
+  const {
+    onBack,
+    onSubmit,
+    submission,
+    renderErrorMessage
+  } = props;
+  let submitButton;
+  let submitMessage;
+
+  if (submission.status === false) {
+    submitButton = (
+      <ProgressButton
+        submitButton
```

Comment:
`submitButton` has been added here to trigger schemaform form page validation logic.